packages/grafana-data/src/types/plugin.ts

```diff
@@ -6,6 +6,7 @@ import { LiveChannelSupport } from './live';
 export enum PluginState {
   alpha = 'alpha', // Only included if `enable_alpha` config option is true
   beta = 'beta', // Will show a warning banner
+  stable = 'stable', // Will show a warning banner
```

Comment:
Will not show?

Reply:
yepp, will not show. Just in case someone put that in plugin.json

Reply:
@torkelo i think he meant that you missed "not" in the comment :)

Reply:
yup -- was on my phone so the suggestion button was awkward ;)

Reply:
Oops :) 
https://github.com/grafana/grafana/commit/7ca66b204004d4d0089de652cd47c51d4195cf36